pkg/gameservers/controller.go

```diff
@@ -83,6 +83,7 @@ type Controller struct {
 	sidecarCPULimit        resource.Quantity
 	sidecarMemoryRequest   resource.Quantity
 	sidecarMemoryLimit     resource.Quantity
+	sidecarSecurityCtx     *corev1.SecurityContext
```

Comment:
If you just drop to the user, this will be simpler, too.

Reply:
True